locales/en/messages.json

```diff
@@ -3599,16 +3599,16 @@
         "description": "Roll pitch ratio slider label"
     },
     "pidTuningRollPitchRatioSliderHelp": {
-        "message": "Roll Pitch Ratio",
-        "description": "This needs a meaningful help text"
+        "message": "The Pitch-Roll ratio is the balance of Moment of Inertia (MoI) between the Pitch and Roll axis.<br><br>Generally, the Pitch axis has a higher MoI and therefore it may be appropriate to have PID gains higher on the Pitch axis vs the Roll axis.  However, on 'Stretched X', or other configurations, this may not be the case.<br><br>Generally for tuning, you are moving up the 'P and D Gain' slider until one axis begins to show D-term oscillation (a 'trilling' noise) and then back it down some.  Blackbox is the easies method to determine which axis is 'trilling'.  Once you find which axis is at the D-term limit, you can use the Pitch-Roll slider to bring the other axis to just short of the same limit to balance out the quad for peak flight performance.",
+        "description": "Pitch-Roll Ratio tuning slider helpicon message"
     },
     "pidTuningIGainSlider": {
         "message": "I Gain:",
         "description": "I gain slider label"
     },
     "pidTuningIGainSliderHelp": {
-        "message": "I Gain",
-        "description": "This needs a meaningful help text"
+        "message": "I-gain needs to be in balance with the P-gain; similar to how the P-gain needs to be in balance with the D-gain (Tuning sequence: Filters->D->P->I).<br><br>The I-term gain sliders adjust I-term while holding P-term constant to adjust this ratio.<br><br>If you are getting slow wobbles when you drop throttle to 0%, that's an idication the I-gain is too high and you should lower the I-gain slider.<br><br>This said, genereally you want the I-term as strong as it can be (but with balance) to keep the quad tracking on the sticks in spirrel turns, orbits, ets...",
```

Comment:
The term used here should be consistent with the term used in `pidTuningIGainSlider` - if not this will be confusing for users.

What does 'ets' at the end stand for?